lib/utils/extractDeviceName.ts

```diff
@@ -0,0 +1,25 @@
+const DEVICE_NAME_REGEXP_1 = new RegExp(/\((.*?)\)/);
+const DEVICE_NAME_REGEXP_2 = /^[0-9A-Fa-f-]{36}\s*(\{.*\})$/;
+
+export default function extractDeviceName(fullDeviceName: string): string {
```

Comment:
The typing could be simplified here and in the extract1, extract2.

Reply:
Simplified